firestore/cloud-client/vector_search_test.py

```diff
@@ -119,9 +119,9 @@ def test_vector_search_distance_result_field_with_mask():
 
     assert len(results) == 4
     assert results[0].to_dict() == {"color": "green", "vector_distance": 0.0}
-    assert results[1].to_dict() == {"color": "blue", "vector_distance": 1.0}
-    assert results[2].to_dict() == {"color": "red", "vector_distance": 7.0}
-    assert results[3].to_dict() == {"color": "red", "vector_distance": 8.0}
+    assert results[1].to_dict() == {"color": "blue", "vector_distance": 0.09812527000000004}
```

Comment:
Very nitpicky, but can we truncate these floating point rounding errors to match the same number of decimal places as the other numbers? Mostly aesthetics :)